libraries/aws/provisioning/src/aws_iot_provisioning_parser.c

```diff
@@ -178,111 +197,90 @@ static AwsIotProvisioningError_t _parseRejectedResponse( IotSerializerDecoderObj
 
 /*------------------------------------------------------------------*/
 
-static AwsIotProvisioningError_t _parseCommonCertInfoInResponse( IotSerializerDecoderObject_t * payloadDecoder,
-                                                                 IotSerializerDecoderObject_t * certPemDecoder,
-                                                                 IotSerializerDecoderObject_t * certIdDecoder,
-                                                                 IotSerializerDecoderObject_t * ownershipTokenDecoder )
+static AwsIotProvisioningError_t _parseKeyedEntryInPayload( IotSerializerDecoderObject_t * payloadDecoder,
+                                                            IotSerializerDecoderObject_t * payloadEntryDecoder,
+                                                            const char * pKeyString,
+                                                            const char * pOperationString )
```

Comment:
The `pOperationString` parameter seems unnecessary if it's going to be `CREATE_KEYS_AND_CERTIFICATE_OPERATION_LOG` every time. Do we plan on this being variable?

Reply:
This common utility is used by 2 different operations (CSR and KeysAndCertificate).
Have pushed the fix.